src/main/java/de/plushnikov/intellij/plugin/provider/LombokAugmentProvider.java

```diff
@@ -46,26 +51,44 @@ public LombokAugmentProvider() {
     modifierProcessors = Arrays.asList(getModifierProcessors());
   }
 
-  @Nullable
+  /**
+   * Support method required by patcher project and {@link ModifierVisibilityClassFileTransformer}.
+   * Provides a simple way to inject modifiers into older versions of IntelliJ. Return of the null value is dictated by legacy IntelliJ API.
+   *
+   * @param modifierList PsiModifierList that is being queried
+   * @param name         String name of the PsiModifier
+   * @return {@code Boolean.TRUE} if modifier exists (explicitly set by modifier transformers of the plugin), {@code null} otherwise.
+   */
+  public Boolean hasModifierProperty(@NotNull PsiModifierList modifierList, @NotNull final String name) {
+
+    final Set<String> modifiers = this.transformModifiers(modifierList, Collections.<String>emptySet());
```

Comment:
I am not really sure (I didn't measured it), but I think we should possible add some cache hier (like it is done in Intellij EAP at the moment), because this function would be called very often...